packages/stark-core/src/modules.ts

```diff
@@ -1,5 +1,6 @@
 export * from "./modules/http";
 export * from "./modules/logging";
+export * from "./modules/error-handler";
```

Comment:
The name of the folder containing the Error Handling module should be `error-handling` instead of `error-handler` ;)